File: teoria_19Agosto/coso 2/Telefono.java
public class Telefono {

    private String marca;
    private String imei;
    private String modelo;
    private String descripcion;
    private Double precio;

    public Telefono(){
        this.marca = "sin marca";
        this.imei = "sin registrar";
    }

    public Telefono(String marca, String imei, String modelo, String descripcion, Double precio){
        this.marca = marca;
        this.imei = imei;
        this.modelo = modelo;
        this.descripcion = descripcion;
        this.precio = precio;
    }

    @Override
    public String toString(){
        return  this.marca + ", " + this.modelo + ", " + this.precio;
    }

}

File: TP2/Python/Ejemplo clase 19/Telefono.py
class Telefono:
    def __init__(self, marca, imei, modelo, descrip, precio):
        self.__marca = marca
        self.__imei = imei
        self.__modelo = modelo
        self.__descrip = descrip
        self.__precio = precio

    def __str__(self) :
        return f'Telefono({self.__marca} {self.__imei} {self.__modelo}{self.__descrip}{self.__precio})'





